### ISSUE TITLE
Browser does not allow backslash as input

### ISSUE THREAD

**Issue by MathemanFlo** [armory3d/armorpaint#1073]
**ArmorPaint version:**
ArmorPaint.org - v0.8 (2021-07-19) - 13ea792

**OS/device including version:**
Win10 x64

**Steps to reproduce:**
1. Open the browser tab
2. Type in C:
3. Type \
4. The backslash is immediately selected, i.e. if you press the next key it replaces the \
5. This behavior can be prevented by pressing arrow right because in this case it is unselected.

**Comment by luboslenco** [comment]
Seems to work ok here, do you still get this?

**Comment by MathemanFlo** [comment]
Yes I do, I tried two independently compiled versions on two Win10 PCs and got the same result. But now I found the issue. It's the keyboard language/layout/locale. The German keyboard has \ on AltGr + - (key next to 0). If I type in \ I get the following: 
![grafik](https://user-images.githubusercontent.com/28649121/128145761-048aca46-4fa9-4583-ba40-14c574ebcf86.png)
I.e. the next character will replace the \ because it is selected.
